lib/hooks/blueprints/actions/create.js

```diff
@@ -40,7 +40,7 @@ module.exports = function createRecord (req, res) {
 				Model.subscribe(req, newInstance);
 				Model.introduce(newInstance);
 			}
-			Model.publishCreate(newInstance.toJSON(), !req.options.mirror && req);
+			Model.publishCreate(newInstance, !req.options.mirror && req);
 		}
```

Comment:
@Hanifb What was the thinking behind this?  Is it somehow necessary to get arrays to work?  By pushing this logic into `publishCreate`, we force users of that method to pass in model instances instead of being able to send in plain objects.  Sorry I didn't catch this earlier.